content/en/continuous_testing/environments/multiple_env.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Testing Multiple Environments
+kind: documentation
+description: Learn how to use Continuous Testing to reuse the same Synthetics scenarios against multiple environments.
+further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-synthetic-ci-cd-testing/"
+  tag: "Blog"
+  text: "Incorporate Datadog Continuous Testing tests into your CI/CD pipeline"
+- link: "https://www.datadoghq.com/blog/internal-application-testing-with-datadog/"
+  tag: "Blog"
+  text: "Test internal applications with Datadog's testing tunnel and private locations"
+- link: "/continuous_testing/environments/proxy_firewall_vpn"
+  tag: "Documentation"
+  text: "Learn about testing while using proxies, firewalls, or VPNs"
+---
+
+## Overview
+
+Continuous Testing allows you to apply the same scenario from scheduled tests against the production environment to development and staging environments. Continuous Testing uses Synthetic tests throughout the development cycle to ensure regressions are caught as soon as possible.
+
+When triggering a CI test, you can overwrite the starting URL of a [browser][1] or [API test][2] to reroute the Synthetic Worker to the appropriate environment.
+
+## Overriding the starting URL
+
+A Synthetic browser test starts the test scenario by navigating to a starting URL. Similarly, an API HTTP test sends a request to a specific URL. When triggering a CI test, you can overwrite this starting URL to point to another environment where your application is deployed in.
+
+{{< img src="continuous_testing/continuous_testing_start-url_substitution.png" alt="Continuous Testing tunnel allows the Synthetics Worker to reach your private applications" width="100%" >}}
+
+When triggering a CI test, the `startUrl` field allows you to overwrite the first URL that a browser test navigates to or the URL used by an HTTP test request. You can specify this option through the global configuration file, the Synthetics configuration files (`*.synthetics.json`), or the command line flag `--start-url`.
+
+```shell
+yarn datadog-ci synthetics run-tests --public-id <public-id> --start-url "https://staging.my-app.com"
```

Comment:
Not everyone uses `yarn`, and the CLI could also be globally installed. I think what's important is the command line, not how `datadog-ci` is access.

One could totally run it like `/path/to/datadog-ci <args>` as well

```suggestion
datadog-ci synthetics run-tests --public-id <public-id> --start-url "https://staging.my-app.com"
```

Reply:
Can you make sure we don't have `yarn datadog-ci` anywhere else?